huh_scriven_velocity.py

```
@@ -31,48 +33,88 @@ def Ux(Theta, Phi):
+    Theta_grid = []
+    Phi_grid = []
+    for theta in Theta:
+        phi_values = np.linspace(0, theta, 100)
+        Theta_grid.extend([theta] * len(phi_values))
+        Phi_grid.extend(phi_values)
+
+    Theta_grid = np.array(Theta_grid)
+    Phi_grid = np.array(Phi_grid)
+
+    # Compute Ur, Uphi, Ux, Uy
+    Ur_grid = Ur(Theta_grid, Phi_grid)
```

Ruff (0.11.9)

Local variable `Ur_grid` is assigned to but never used

Remove assignment to unused variable `Ur_grid`

(F841)